java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -770,7 +774,7 @@ void testFromScalarZeroRows() {
         }
 
         try (ColumnVector c = ColumnVector.fromScalar(s, 0)) {
-          assertEquals(type, c.getType());
+          assertEquals(type, c.getType().typeId);
```

Comment:
Why not do all of this in terms of DType instead of DTypeEnum so we get the scale check in here too.

Reply:
Sorry, there wasn't easy way to get all the types for test here. Will try to do in in terms of DType.

Reply:
This is resolved by @sperlingxx's PR which is merged to this PR.